cli/src/pcluster/validators/cluster.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
in a next patch I'd rename all the validators modules by adding `_validators` prefix.

validators/cluster.py --> validators/cluster_validators.py
etc